KarhooUISDK/CommonUI/Views/KarhooMaterialDesignTextField.swift

```diff
@@ -7,60 +7,160 @@
 //
 
 import SwiftUI
+import Combine
 
 struct KarhooMaterialDesignTextField: View {
```

Comment:
Could you create a ViewModel which contains all data like `title`, `mainPlaceholder`, `isMandatory`, `contentInputType` etc? It'd simplify the type a lot.